### PR TITLE
Use pbr library for versioning

### PR DESCRIPTION
**Pull request by dblanchette**
Searching around for a solution to my versioning woes, I found this library. It reads the version from the git tag. With my other PR that will add a publish to pypi on commit, we just need to push to master and add the version tag to publish.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = json-schema-for-humans
+description = Generate static HTML documentation from JSON schemas
+description-file = README.md
+author = Denis Blanchette, AbdulRahman Al Hamali
+author-email = tools@coveo.com
+license = Apache Software License
```
Comment (AbdulRahmanAlHamali) [inline setup.cfg]: V2

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+url = https://github.com/coveooss/json-schema-for-humans
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    License :: OSI Approved :: Apache Software License
```
Comment (AbdulRahmanAlHamali) [inline setup.cfg]: V2

**Comment by dblanchette** [comment]
@julienduchesne In my other PR I added a GitHub action to launch run `python setup.py sdist bdist_wheel` on adding tags. pbr will use the tag automagically.

File: setup.cfg (hunk 2 above)
Comment (dblanchette) [reply setup.cfg]: I thought so too, but I am only using those that are listed here: https://pypi.org/classifiers/

**Pull request merged**